rules/gke_rules.yaml

```diff
@@ -51,8 +51,8 @@ rules:
     check_serverconfig_valid_master_versions: false
 
 # This rule checks for any node pools that are running a version
-# that is not patched for Meltdown & Spectre.
-  - name: Meltdown and Spectre patches not applied
+# that is not patched for current known and patched critical CVEs.
```

Comment:
Is there a way to somehow retain some mentioning of ```Meltdown & Spectre``` as specific examples for users?